helm/hpcc/values.schema.json

```diff
@@ -179,6 +179,10 @@
           "type": "string",
           "default": "hpccsystems"
         },
+        "name": {
```

Comment:
@richardkchapman this is the change I mentioned, allowing  global.image.name to be set.

Reply:
Looks correct